### DISCUSSION TITLE
[FEAT] Labelling tool - Hide Splink predictions from the page by default

### DISCUSSION THREAD

**Discussion by medwar99** [moj-analytical-services/splink#2526] [Ideas]
### Is your proposal related to a problem?
The labelling tool defaults to showing the Splink predictions by default, which is acknowledged as potentially introducing bias. We can currently tackle this issue in two ways: 
* Completely remove the ability to see the predictions by setting `show_splink_predictions_in_interface=False`.
* Toggle the view to off by using the checkbox on the page.

The first option is good, and I believe wants to stay. It may not be suitable to allow reviewers even the option to see the predictions, but these tools tend to be for linkage operators and so defaulting `show_splink_predictions_in_interface` to True in in Python land works. 

The problem with the second option is that the default behaviour of the tool is to show the predictions when loaded and so users may see Splink predictions before being able to toggle the view off. The default value setting returned by the `_show_splink_predictions_initial_value()` function of [labelling_tool/slt.js](https://github.com/moj-analytical-services/splink/tree/master/splink/internals/files/labelling_tool/slt.js) sets a value of "Show splink predictions" if SHOW_SPLINK_PREDICTIONS is not false, causing the view to be visible on page load. 

https://github.com/moj-analytical-services/splink/blob/2aa78da2029f300fce56e76e73bbffe35087ae25/splink/internals/files/labelling_tool/slt.js#L8823-L8832

### Describe the solution you'd like
Would it be better to have this checkbox default to be off on page load, and then allow users to toggle the checkbox if they want to see the Splink predictions? I updated the `_show_splink_predictions_initial_value` function to the following, which seemed to act as intended:

```
 function _show_splink_predictions_initial_value(globalThis)
  {
    if (typeof globalThis.SHOW_SPLINK_PREDICTIONS == "undefined") {
      return [];
    } else if (globalThis.SHOW_SPLINK_PREDICTIONS == true) {
      return [];
    } else {
      return [];
    }
  }
```

My JS skills aren't really good enough to know for sure if a) it has wider implications and b) makes the function a bit redundant.

### Describe alternatives you've considered
* I tried closing my eyes and clicking the checkbox before I looked at the page, but sometimes misclicked.
* I tried clicking really really quickly, but unfortunately I'm not that fast. 

In seriousness, I considered whether changing the `show_splink_predictions_in_interface` arg of `linker.labelling_tool_for_specific_record()` from a Boolean to a string might be worthwhile, but this seems unnecessarily complicated.

**Comment by RobinL** [comment]
Agreed - this seems sensible to me too.  I think probably having three options rather than a bool is fine here.  We can use hinting to make it autocomplete for users.  I've converted into an issue for when we get round it it.  It's a bit fiddly because, whilst you can directly edit the js, those files are the product of a wider js compilation process, so to do it 'properly' the changes have to be made upstream in other code that's not very well organised.
